nilearn/maskers/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,263 @@
+                       sample_mask=None):
+        """Extract signals from a list of 4D niimgs.
+
+        Parameters
+        ----------
+        imgs_list: list of 4D Niimg-like objects
+            See http://nilearn.github.io/manipulating_images/input_output.html
+            Images to process. Each element of the list is a 4D image.
+
+        confounds: CSV file or array-like, optional
+            This parameter is passed to signal.clean. Please see the related
+            documentation for details.
+            shape: list of (number of scans, number of confounds)
+
+        sample_mask : Any type compatible with numpy-array indexing, optional
+            shape: (number of scans - number of volumes removed, )
+            Masks the niimgs along time/fourth dimension to perform scrubbing
+            (remove volumes with high motion) and/or non-steady-state volumes.
+            This parameter is passed to signal.clean.
+
+                .. versionadded:: 0.8.0
+
+        Returns
+        -------
+        region_signals: list of 2D numpy.ndarray
+            List of signals for each label per subject.
+            shape: list of (number of scans, number of labels)
+
+        """
+        # We handle the resampling of labels separately because the affine of
+        # the labels image should not impact the extraction of the signal.
+
+        self._check_fitted()
+        niimg_iter = _iter_check_niimg(imgs_list, ensure_ndim=None,
+                                       atleast_4d=False,
+                                       memory=self.memory,
+                                       memory_level=self.memory_level,
+                                       verbose=self.verbose)
+
+        if confounds is None:
+            confounds = itertools.repeat(None, len(imgs_list))
+
+        func = self._cache(self.transform_single_imgs)
+
+        region_signals = Parallel(n_jobs=n_jobs)(
+            delayed(func)(imgs=imgs, confounds=cfs, sample_mask=sample_mask)
+            for imgs, cfs in zip(niimg_iter, confounds))
+        return region_signals
+
+    def transform(self, imgs, confounds=None, sample_mask=None):
+        """ Apply mask, spatial and temporal preprocessing
+
+        Parameters
+        ----------
+        imgs: list of Niimg-like objects
+            See http://nilearn.github.io/manipulating_images/input_output.html
+            Data to be preprocessed
+
+        confounds: CSV file path or 2D matrix
+            This parameter is passed to signal.clean. Please see the
+            corresponding documentation for details.
+
+        sample_mask : Any type compatible with numpy-array indexing, optional
+            shape: (number of scans - number of volumes removed, )
+            Masks the niimgs along time/fourth dimension to perform scrubbing
+            (remove volumes with high motion) and/or non-steady-state volumes.
+            This parameter is passed to signal.clean.
+
+                .. versionadded:: 0.8.0
```

Comment:
to be updated

Reply:
the version added should be 0.9.2

Reply:
I was a bit unsure because to me it seems like the versionadded is referring specifically to when the `sample_mask` argument was added to the function `transform` so I am not sure it makes sense to put the current version. But also maybe it makes sense to remove the versionadded line entirely since it is a 'new' function in a new class?

Reply:
Then remove the thing indeed.